src/main.cpp

```diff
@@ -381,8 +439,20 @@ init_create_parser(CLI::App* subcom)
     });
 }
 
-std::string
-version()
+void
+init_list_parser(CLI::App* subcom)
+{
+    subcom->add_option("specs", create_options.specs, "Specs to install into the new environment");
+    init_network_parser(subcom);
```

Comment:
do we need this? I thought we're printing the local packages here so I guess no need for network options?